src/classBased/components/InputTodo.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+import Button from './Button'
+import Input from './Input'
+import { FaPlusCircle} from 'react-icons/fa'
+
+class InputTodo extends Component {
+	state = {
+		title: '',
+	}
```

Comment:
- [ ] As the previous reviewer has earlier highlighted, I think it will be better to initialize your state object in a constructor () and also call the super() method to inherit from the react component parent class 👍 because the constructor method is a special method for creating and initializing an object created with a class so it is important to use it 😄 

e.g 
```
class InputTodo extends Component {
    constructor() {
       super()
      this.state = { 
         title: ' ',
      }
   }
...
}
```